JavaSource/org/unitime/timetable/webutil/WebInstructionalOfferingTableBuilder.java

```diff
@@ -1798,9 +1808,12 @@ protected void buildConfigRow(Vector subpartIds, ClassAssignmentProxy classAssig
                     row.addContent(this.buildExamRoom(examAssignment, exams, isEditable));
                 }
             }
-	    		if (isShowLms()) {
-	                row.addContent(initNormalCell("", isEditable));			
-	    		}
+    		if (isShowLms()) {
+                row.addContent(initNormalCell("", isEditable));			
+    		}
+    		if (isShowFundingDepartment()) {
```

Comment:
Funding department column was already added, just after the Manager. See line 1745.